src/gpio_messaging.toit

```diff
@@ -0,0 +1,38 @@
+import serialization
+
+/**
```

Comment:
It's a bit more minimal than before, but lots of code wasn't necessary, as null values were already caught earlier.

Reply:
Thanks for clearing out the nullable - was good to know that that is not necessary. I usually program c#, and even though a property is not nullable there are ways of getting around this. 

Besides that, it is very nice to see different ways to work with Maps (initializing and getting). :)